### PR TITLE
Bump Typescript version to 4.9.3

### PR DESCRIPTION
**Pull request by fpdrozd**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "oclif": "^3.0.1",
     "ts-jest": "^28.0.5",
     "ts-node": "^10.7.0",
-    "typescript": "^4.6.4"
+    "typescript": "4.9.3"
```
Comment (kshalot) [inline package.json:55]:
```suggestion
    "typescript": "^4.9.3"
```

Or was this intentional?

Comment (fpdrozd) [reply package.json]: Right, thanks!

**Review by kshalot — APPROVED** [review]

**Comment by kshalot** [comment]
Also one comment for the future - I'd opt for creating separate PRs for non-major (not breaking) dependency bumps and code changes resulting from them

**Comment by fpdrozd** [comment]
@kshalot you mean my change with the if statement? - Yeah, I think it's a good practice to separate them.

**Comment by kshalot** [comment]
> @kshalot you mean my change with the if statement? - Yeah, I think it's a good practice to separate them.

Yes, that's what I meant.

**Pull request merged**